extension/src/popup/views/__tests__/ManageAssets.test.tsx

```diff
@@ -390,7 +390,7 @@ describe.skip("Manage assets", () => {
     );
   });
 
-  it.only("renders manage assets view initial state", async () => {
+  it("renders manage assets view initial state", async () => {
```

Comment:
this is an unrelated change - I noticed that all the other tests in this file were not running. All of these tests pass, so I think this may have been an oversight